lib/ramble/spack/util/debug.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

Comment:
Is it OK to update the copyright for this? IMO, it should stay the same as when we vendored spack in, but I could be wrong about that.

Reply:
I left it like that since I took the updated it directly from the current spack. I think it's a good question. To me keeping it like this was better because:

1. It's true to the file I bought in
2. It's a nice marker to us ramble folks that it's already updated relative to the rest of `./spack`

I'm happy to do whatever you think is best, I just felt weird about downgrading the header such that it mismatches the contents of the file

Reply:
By luck there was an old version with the file contents we want and matching header. Going back to that to avoid any accidental headaches